packages/outline-view/src/browser/outline-view-widget.tsx

```diff
@@ -50,7 +51,7 @@ export class OutlineViewWidget extends TreeWidget {
 
     constructor(
         @inject(TreeProps) protected readonly treeProps: TreeProps,
-        @inject(TreeModel) model: TreeModel,
+        @inject(OutlineViewTreeModel) model: OutlineViewTreeModel,
```

Comment:
I think this might require a breaking change note in the changelog.

Reply:
I think it was not a breaking change, but please correct me if I am wrong. `TreeModel` is a supertype of the `OutlineViewTreeModel`.

Something like this:
```ts
class Base {
    foo(): void {
    }
}

class Sub extends Base {
}

class Lib {
    constructor(base: Base) {
        base.foo();
    }
}

class Client extends Lib {
    constructor(base: Base) {
        super(base);
    }
}
```

If I change the `ctor` of `Lib` and have `base: Sub` instead, my client does not have any API changes.